charts/posthog/values.yaml

```diff
@@ -643,6 +643,42 @@ serviceAccount:
   # -- Configures annotation for the ServiceAccount
   annotations: {}
 
+
+###
+###
+### ---- Grafana ----
+###
+###
+grafana:
+  # -- Whether to install Grafana or not.
+  enabled: false
+
+  # -- Sidecar configuration to automagically pull the dashboards from the `charts/posthog/grafana-dashboard` folder. See [official docs](http://docs.grafana.org/administration/provisioning/#dashboards) for more info.
```

Comment:
While these doc links do include useful information, it seems like https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md would be more useful.